packages/experimental/Expander/just.config.js

```diff
@@ -0,0 +1,3 @@
+const { preset } = require('@uifabricshared/build-native');
+
```

Comment:
Is this file something all native FURN components need?

Reply:
It's something that the initial components had, and I wrote my contribution guide to say "Copy all the files in an existing package and edit as appropriate". It's likely we need it, though I'm not sure why.

Reply:
This defines tasks like linting and building.